src/Menu.js

```diff
@@ -76,7 +76,7 @@ module.exports = function (mainWindow, app, focusOrCreateWindow, updater, log) {
     }, {
       label: 'Copy Debug Info',
       click () {
-        fetch(`http://127.0.0.1:37370/debugger`, {
+        window.fetch(`http://127.0.0.1:37370/debugger`, {
```

Comment:
It might be better to make fetch an allowed global, in any case that window is undefined